src/pages/Governance/index.spec.tsx

```diff
@@ -268,4 +285,60 @@ describe('pages/Governance', () => {
       routes.governanceProposal.path.replace(':proposalId', '98'),
     );
   });
+
+  it('shows the create proposal option on BSC_TESTNET', async () => {
+    (useAuth as Vi.Mock).mockImplementation(() => ({
+      accountAddress: fakeAccountAddress,
+      chainId: ChainId.BSC_TESTNET,
+    }));
+    const { queryAllByTestId } = renderComponent(<Governance />, {
+      routerOpts: {
+        routerInitialEntries: ['/governance/proposal-create', '/governance'],
+        routePath: '/governance/*',
+      },
+    });
+    expect(queryAllByTestId(TEST_IDS.createProposal)).toHaveLength(1);
+  });
+
+  it('shows the create proposal option on BSC_MAINNET', async () => {
+    (useAuth as Vi.Mock).mockImplementation(() => ({
+      accountAddress: fakeAccountAddress,
+      chainId: ChainId.BSC_MAINNET,
+    }));
+    const { queryAllByTestId } = renderComponent(<Governance />, {
+      routerOpts: {
+        routerInitialEntries: ['/governance/proposal-create', '/governance'],
+        routePath: '/governance/*',
+      },
+    });
+    expect(queryAllByTestId(TEST_IDS.createProposal)).toHaveLength(1);
+  });
+
+  it('hides the create proposal option on ETHEREUM', async () => {
+    (useAuth as Vi.Mock).mockImplementation(() => ({
+      accountAddress: fakeAccountAddress,
+      chainId: ChainId.ETHEREUM,
+    }));
+    const { queryAllByTestId } = renderComponent(<Governance />, {
+      routerOpts: {
+        routerInitialEntries: ['/governance/proposal-create', '/governance'],
+        routePath: '/governance/*',
+      },
+    });
+    expect(queryAllByTestId(TEST_IDS.createProposal)).toHaveLength(0);
+  });
+
+  it('hides the create proposal option on SEPOLIA', async () => {
+    (useAuth as Vi.Mock).mockImplementation(() => ({
+      accountAddress: fakeAccountAddress,
+      chainId: ChainId.SEPOLIA,
+    }));
+    const { queryAllByTestId } = renderComponent(<Governance />, {
+      routerOpts: {
+        routerInitialEntries: ['/governance/proposal-create', '/governance'],
+        routePath: '/governance/*',
+      },
+    });
+    expect(queryAllByTestId(TEST_IDS.createProposal)).toHaveLength(0);
```

Comment:
Nit: you should be able to retrieve the button using its text:
```tsx
getByText(en.vote.createProposalPlus)
```

Reply:
Using `getByText` or `getByTestId` in the test cases where it won't be rendered will throw an error, so I opted to add an ID to the container and query any instances of it in the document.